monkey/monkey_island/cc/ui/src/services/AuthService.js

```diff
@@ -48,12 +96,15 @@ export default class AuthService {
       })
     }).then(response => response.json())
       .then(res => {
-        let token = this._getTokenFromResponse(res);
-        if (token !== undefined) {
-          this._setToken(token);
+        let authToken = this._getTokenFromResponse(this.AUTH_TOKEN_NAME_IN_RESPONSE, res);
+        let refreshToken = this._getTokenFromResponse(this.REFRESH_TOKEN_NAME_IN_RESPONSE, res);
+        if (authToken !== undefined && refreshToken !== undefined) {
+          this._setToken(this.AUTH_TOKEN_NAME_IN_LOCALSTORAGE, authToken);
+          this._setToken(this.REFRESH_TOKEN_NAME_IN_LOCALSTORAGE, refreshToken);
```

Comment:
This is duplicated in a couple of places. Let's add something like `setTokensInLocalStorage(authToken, refreshToken)`.